test/integration/quota/quota_test.go

```diff
@@ -449,6 +476,47 @@ func TestQuotaLimitService(t *testing.T) {
 	_, _, closeFn := framework.RunAnAPIServerUsingServer(controlPlaneConfig, s, h)
 	defer closeFn()
 
+	var subjects []flowcontrol.Subject
+	subjects = append(subjects, flowcontrol.Subject{
+		Kind: flowcontrol.SubjectKindUser,
+		User: &flowcontrol.UserSubject{
+			Name: "system:unsecured",
+		},
+	})
+	_, err = clientset.FlowcontrolV1beta2().FlowSchemas().Create(
+		context.TODO(),
+		&flowcontrol.FlowSchema{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: "testing-exempt-flowschema",
+			},
+			Spec: flowcontrol.FlowSchemaSpec{
+				MatchingPrecedence: 2,
+				PriorityLevelConfiguration: flowcontrol.PriorityLevelConfigurationReference{
+					Name: "exempt",
+				},
+				DistinguisherMethod: &flowcontrol.FlowDistinguisherMethod{
+					Type: flowcontrol.FlowDistinguisherMethodByUserType,
+				},
+				Rules: []flowcontrol.PolicyRulesWithSubjects{
+					{
+						Subjects: subjects,
+						NonResourceRules: []flowcontrol.NonResourcePolicyRule{
+							{
+								Verbs:           []string{flowcontrol.VerbAll},
+								NonResourceURLs: []string{flowcontrol.NonResourceAll},
+							},
+						},
+					},
+				},
+			},
+		},
+		metav1.CreateOptions{})
+	if err != nil {
+		t.Errorf("creating flow schema should not have returned error: %v", err)
+	}
+
+	waitForFlowSchema(t, clientset, "testing-exempt-flowschema")
+
```

Comment:
if we're running with the system privileged group, the existing default `exempt` flowschema should already exempt us from APF, right? I don't think we need these lines

Reply:
Originally I planned to use existing default exempt flowschema instead create a new one, but when I tried to correlate the new create user "system:unsecured" with  default exempt flowschema, it failed indicating exempt flowschema can not be modified.

So, either we should add a new flowschema, or use existing default exempt flowschema with the user "system:masters"

Reply:
the existing exempt flowschema selects requests from the privileged group, which this test runs with now, right?

Reply:
```go
	MandatoryFlowSchemaExempt = newFlowSchema(
		"exempt",
		flowcontrol.PriorityLevelConfigurationNameExempt,
		1,  // matchingPrecedence
		"", // distinguisherMethodType
		flowcontrol.PolicyRulesWithSubjects{
			Subjects: groups(user.SystemPrivilegedGroup),
...
```

Reply:
Yeah, as you mentioned, we do not need add another excmpt flowschema. These lines have been removed.

I have run the test again and the output shows APF is selecting that expmpt flowschema as expected.

And one hour stress test has paased as well. Could you please take another look, thank you